packages/atomic/src/components/atomic-query-summary-v1/atomic-query-summary.tsx

```diff
@@ -0,0 +1,109 @@
+import {Component, h, State} from '@stencil/core';
+import {
+  QuerySummary,
+  QuerySummaryState,
+  buildQuerySummary,
+} from '@coveo/headless';
+import {
+  Bindings,
+  BindStateToController,
+  InitializableComponent,
+  InitializeBindings,
+} from '../../utils/initialization-utils';
+import escape from 'escape-html';
+
+/**
+ * The `atomic-query-summary` component displays information about the current range of results and the request duration (e.g., "Results 1-10 of 123 in 0.47 seconds").
+ *
+ * @part container - The container for the whole summary.
+ * @part results - The container for the results.
+ * @part no-results - The container for when there are no results.
+ * @part highlight - The summary highlights.
+ * @part placeholder - The query summary placeholder used while the search interface is initializing.
+ */
+@Component({
+  //TODO remove v1
+  tag: 'atomic-query-summary-v1',
```

Comment:
If i'm not mistaken, this component is pretty much the same as the original query summary component?

Can we just do these 2 things in `atomic-query-summary/atomic-query-summary.tsx`:
- set a TODO on line 45 to set the `enableDuration` prop to false by default (will still be requested by customers Im sure)
- set a TODO on line 66/124 to remove the `renderNoResults` method

Reply:
Yes they are pretty much the same. I assumed we were removing the duration since it wasn't included in the designs but if we keep it then they are identical except for not rendering when there are no results.

Reply:
I will remove all the v1 files and just add the TODOs